src/main/java/uk/gov/hmcts/ccd/v2/V2.java

```diff
@@ -13,9 +13,9 @@
 
         // Internal API
         public static final String UI_CASE_VIEW = "application/vnd.uk.gov.hmcts.ccd-data-store-api.ui-case-view.v2+json;charset=UTF-8";
-        public static final String UI_START_TRIGGER = "application/vnd.uk.gov.hmcts.ccd-data-store-api.ui-start-trigger.v2+json;charset=UTF-8";
```

Comment:
We can deprecate this now since FE has moved on to using the correct VND for start case trigger endpoint.